torch/nested/_internal/ops.py

```diff
@@ -897,16 +894,38 @@ def sum_dim_IntList(func, *args, **kwargs):
                     "sum(): not supported along a ragged and non-batch dimension for NestedTensor"
                 )
             # reduction cases: (ragged)
+            # shift reduction dimension of values backward to outer dimension
+            values_transposed_backward = inp._values.permute(
```

Comment:
nit: can we use a different name other than "backward" to avoid getting confused with "backward" as in "forward/backward pass"

e.g. maybe "values_standard_layout" or "values_ragged_outer"

Reply:
yeah that sounds good! changed the `backward` and `forward` variable names